{{cookiecutter.project_slug}}/overlays/dlp_proxy/training-infra/templates/dlp-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dlp-nginx-proxy
+  namespace: minio-external-{% raw %}{{ .Values.environment }}{% endraw %}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dlp-nginx-proxy
+  template:
+    metadata:
+      labels:
+        app: dlp-nginx-proxy
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:alpine
+        ports:
+        - containerPort: 9000
+        volumeMounts:
+        - name: nginx-config
+          mountPath: /etc/nginx/nginx.conf
+          subPath: nginx.conf
+        - name: nginx-logs
+          mountPath: /var/log/nginx
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "100m"
+          requests:
+            memory: "64Mi"
+            cpu: "50m"
```

Comment:
This is quite a lot for an NGINX proxy, particularly in our test scenario where it's seldomly used. Does the Kind cluster actually honor these requests and limits? If so, we should probably remove/reduce them.